main/abuild/0001-abuild-properly-detect-package-size-on-filesystems.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/abuild.in b/abuild.in
+index 35b7995572..2f68cb581e 100644
+--- a/abuild.in
++++ b/abuild.in
+@@ -812,6 +812,14 @@ prepare_metafiles() {
+ 	cd "$dir"
+ 	mkdir -p "$controldir"
+ 	local builddate=$(date -u "+%s")
++	
++	# Fix package size on several filesystems
++        case "$(df -PT . | awk '{print $2}' | tail -n1)" in
++                btrfs|ecryptfs|zfs)
++                sync;;
++                *);;
```

Comment:
correct, but I like neat programming, just a personal style, default case will be removed